src/iracingdataapi/client.py

```diff
@@ -125,51 +125,236 @@ def series(self):
+
+                [
+                    {"label": "ALL", "value": -1},
+                    {"label": "Division 1", "value": 0},
+                    {"label": "Division 2", "value": 1},
+                    //...
+                    {"label": "Rookie", "value": 10}
+                ]
+        """
         return self._get_resource("/data/constants/divisions")
 
     def constants_event_types(self):
+        """Fetches a list containing the event types.
+
+        Retrieves a list containing the types of events (i.e. Practice,
+        Qualify, Time Trial or Race)
+
+        Returns:
+            A list of dicts representing each event type.
+
+        Example:
+            ::
+
+                [
+                    {"label": "Practice", "value": 2},
+                    {"label": "Qualify", "value": 3},
+                    {"label": "Time Trial", "value": 4},
+                    {"label": "Race", "value": 5}
+                ]
+        """
         return self._get_resource("/data/constants/event_types")
 
     def get_cars(self):
+        """Fetches a list containing all the cars in the service.
+
+        Retrieves a list containing each car in the service, with
+        detailed information related to the car within iRacing.
+
+        Returns:
+            list: A list of dicts representing each car information.
+        """
         return self._get_resource("/data/car/get")
 
     def get_cars_assets(self):
+        """Fetches a list containing all the car assets in the service.
+
+        Retrieves a list containing each car in the service, with
+        detailed assets related to the car (i.e. photos, tech specs, etc.)
+
+        Returns:
+            list: A list of dicts representing assets from each car.
+        """
         return self._get_resource("/data/car/assets")
 
     def get_carclasses(self):
+        """Fetches a list containing all the car classes in the service.
+
+        Retrieves a list containing each car class in the service, including
+        all the cars in class and basic information of the car class.
+
+        Returns:
+            list: A list of dicts representing assets from each car.
+
+        Example:
+            ::
+
+                [
+                    {
+                        'car_class_id': 4020,
+                        'cars_in_class': [
+                            {'car_dirpath': 'hyundaielantracn7', 'car_id': 146, 'retired': False},
+                            {'car_dirpath': 'hondacivictyper', 'car_id': 147, 'retired': False}
+                        ],
+                        'cust_id': 0,
+                        'name': 'Crev23',
+                        'relative_speed': 45,
+                        'short_name': 'Crev23'
+                    }
+                    // And +100 more...
+                ]
+        """
         return self._get_resource("/data/carclass/get")
 
     def get_carclass(self):
+        """Fetches a list containing all the car classes in the service.
```

Comment:
I'm wondering whether it's worth ditching the docstring here, or replacing it with the deprecation notice?

Reply:
Yes, you're right. Fixed!